documentation/modules/rn-2.6.0.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere, {rhv-full}, or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+
+This release has the following technical changes:
+
+.Technical change
+
+Technical change
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.Cross-Provider Features
+
+Cross-provider features. link:https://issues.redhat.com/browse/MTV-802[(MTV-802)]
+
+.RH-SDL
+
+RH-SDL. link:https://issues.redhat.com/browse/MTV-801[(MTV-801)]
+
+.OCP Provider
+
+OCP (OpenShift Container Platform) provider. link:https://issues.redhat.com/browse/MTV-739[(MTV-739)]
+
+.Console Plugin
+
+Console plugin. link:https://issues.redhat.com/browse/MTV-736[(MTV-736)]
+
+.OVA Provider
+
+OVA provider. link:https://issues.redhat.com/browse/MTV-690[(MTV-690)]
+
+[id="known-issues-26_{context}"]
+== Known issues
```

Comment:
we should keep the following ones from 2.5.5:
Deleting migration plan does not remove temporary resources
Unclear error status message for VM with no operating system
Migration of virtual machines with encrypted partitions fails during conversion (unfortunately)
Migration fails during precopy/cutover while a snapshot operation is performed on the source VM
Unable to schedule migrated VM with multiple disks to more than one storage classes of type hostPath
Non-supported guest operating systems in warm migrations
VMs from vSphere with RHEL 9 guest operating system may start with network interfaces that are down

The only item that should be dropped from the known issues and move to the resolved issues is:
Import OVA: ConnectionTestFailed message appears when adding OVA provider